src/config/constants.ts

```diff
@@ -42,4 +43,8 @@ export const MERKLE_CLAIM_BUCKET = process.env.MERKLE_CLAIM_BUCKET || 'MISSING R
 // third party api constants
 export const CURVE_API_URL = 'https://www.curve.fi/raw-stats/apys.json';
 export const COINGECKO_URL = 'https://api.coingecko.com/api/v3/simple';
-export const ETHERS_JSONRPC_PROVIDER = new ethers.providers.JsonRpcProvider('https://cloudflare-eth.com/');
+
+export enum Provider {
```

Comment:
I think this should be moved to a config service at some point. Doesn't need to be in this PR, but just writing it down.